packages/hash/frontend/src/components/hooks/useReorderPage.ts

```diff
@@ -7,7 +7,7 @@ import {
 import { getAccountPagesTree } from "../../graphql/queries/account.queries";
 import { setParentPage } from "../../graphql/queries/page.queries";
 
-export const useReorderPage = (accountId: string) => {
+export const useReorderPage = (_ownedById: string) => {
```

Comment:
Why are we even taking this here 🤔 Can we remove it now without too many changes or not really?

Reply:
Not exactly sure why it was added, have removed it in https://github.com/hashintel/hash/pull/1157/commits/366706cb09f84ce5d9edf682a3b9d1e5a220cf6a

Reply:
I had changed the parameter name when it was unused, should have been removed instead, thanks!